modules/apps/transfer/keeper/migrations.go

```diff
@@ -105,3 +110,50 @@ func (m Migrator) MigrateTotalEscrowForDenom(ctx sdk.Context) error {
 func equalTraces(dtA, dtB types.DenomTrace) bool {
 	return dtA.BaseDenom == dtB.BaseDenom && dtA.Path == dtB.Path
 }
+
+// setDenomTrace sets a new {trace hash -> denom trace} pair to the store.
+func (k Keeper) setDenomTrace(ctx sdk.Context, denomTrace types.DenomTrace) {
```

Comment:
copy/pasted existing implementations

Reply:
I guess we don't have a test for these :eyes:

Reply:
:eyes: